sample-app/src/components/StaticFilters.tsx

```diff
@@ -65,21 +71,64 @@ export default function StaticFilters(props: StaticFiltersProps): JSX.Element {
   }
 
   return (
-    <fieldset>
-      <legend>{title}</legend>
-      <div>
-        {options.map((option, index) => (
-          <div key={index}>
-            <CheckboxFilter
-              fieldId={option.fieldId}
-              value={option.value}
-              label={option.label}
-              selected={getOptionSelectStatus(option)}
-              optionHandler={handleFilterOptionChange}
-            />
+    <div className={cssClasses.container}>
+      {filterConfig.map((filterSet, index) => {
+        const isLastFilterSet = index === filterConfig.length - 1;
+        return <fieldset>
+          <legend className={cssClasses.title}>{filterSet.title}</legend>
+          <div className={cssClasses.optionsContainer}>
+            {filterSet.options.map((option, index) => (
+              <div key={index}>
```

Comment:
I think we can remove this div? and move `key={index}` into CheckboxFilter tag `<CheckboxFilter key={index} .../>`

Reply:
yep, updated!